chore(docs): Migrate the callouts to bullets (#3580)
kroxylicious/kroxylicious: kroxylicious-docs/docs/_modules/configuring/con-configuring-toplevel-other-settings.adoc

----
# ... (filterDefinitions, virtualClusters etc)
management:
  bindAddress: 0.0.0.0
  port: 9190
  endpoints:
    prometheus: {}
----
where:

* `bindAddress` is the address the HTTP server should bind to. Defaults to `0.0.0.0`.

> * `port` is the port the HTTP server should bind to. Defaults to `9190`.

Reviewer: ```suggestion
* `port` specifies the port the HTTP server binds to. Defaults to `9190`.
```

* `endpoints` defines the services exposed by the management interface.

> ** `prometheus` If present and not null, exposes a Prometheus scrape endpoint at path `/metrics`.

Reviewer: ```suggestion
** `prometheus`, if present and not null, exposes a Prometheus scrape endpoint at `/metrics`.
```
